src/client/core/utils/dom.js

```diff
@@ -400,6 +400,17 @@ export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
     return null;
 }
 
+export function getNodeTypeDescription (node) {
+    var nodeType = node.nodeType;
+
+    for (var property in Node) {
+        if (Node.hasOwnProperty(property)) {
+            if (Node[property] === nodeType)
+                return property.toLowerCase().replace('_', ' ');
```

Comment:
It's based on `Node type constants` https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#Node_type_constants